tests/tests_cib_constraints_create.yml

```diff
@@ -0,0 +1,626 @@
+      - resource_first:
+          id: d2
+          action: demote
+        resource_then:
+          id: d4
+          action: stop
+        id: co-all
+        options:
+          - name: score
+            value: 10
+          - name: symmetrical
+            value: "false"
+          - name: require-all
+            value: "false"
+      # simple set constraint
+      - resource_sets:
+          - resource_ids:
+              - d1
+              - d2
+      # complex set constraint
+      - resource_sets:
+          - resource_ids:
+              - d1
+              - d2
+            options:
+              - name: action
+                value: promote
+          - resource_ids:
+              - d3
+              - d4
+            options:
+              - name: sequential
+                value: "false"
+              - name: require-all
+                value: "true"
+        id: co-set
+        options:
+          - name: symmetrical
+            value: "false"
+          - name: kind
+            value: Serialize
+    ha_cluster_constraints_ticket:
+      # simplest constraint
+      - resource:
+          id: d1
+        ticket: ticket1
+      # constraint id specified
+      - resource:
+          id: d1
+        id: ct-id
+        ticket: ticket2
+      # role specified
+      - resource:
+          id: d2
+          role: Promoted
+        ticket: ticket1
+      # options specified
+      - resource:
+          id: d2
+        ticket: ticket2
+        options:
+          - name: loss-policy
+            value: stop
+      # everything specified
+      - resource:
+          id: d3
+          role: Unpromoted
+        id: ct-all
+        ticket: ticket3
+        options:
+          - name: loss-policy
+            value: demote
+      # simple set constraint
+      - resource_sets:
+          - resource_ids:
+              - d1
+        ticket: ticket-set1
+      # complex set constraint
+      - resource_sets:
+          - resource_ids:
+              - d1
+              - d2
+          - resource_ids:
+              - d3
+              - d4
+            options:
+              - name: sequential
+                value: "false"
+              - name: require-all
+                value: "true"
+        id: ct-set
+        ticket: ticket-set1
+        options:
+          - name: loss-policy
+            value: fence
+
+  tasks:
+    - block:
+        - name: Set up test environment
+          include_tasks: tasks/setup_test.yml
+
+        - name: Run HA Cluster role
+          include_role:
+            name: linux-system-roles.ha_cluster
+
+        - name: Fetch cluster versions of cluster components
+          include_tasks: tasks/fetch_versions.yml
+
+        # yamllint disable rule:line-length
+        - block:
+            - name: Fetch location constraints configuration from the cluster
+              command:
+                cmd: pcs constraint location --full
+              register: __test_pcs_location_config
```

Comment:
`changed_when: false`

Reply:
might need to add this to several other `command` uses in tests

Reply:
I added `changed_when: no` to (hopefully) all 'fetch configuration' commands in tests.